lib/components/user/user-account-screen.js

```diff
@@ -122,34 +127,26 @@ class UserAccountScreen extends Component {
             // and to its own blur/change/submit event handlers that automate the state.
             // We pass the Formik props below to the components rendered so that individual controls
             // can be wired to be managed by Formik.
-            (formikProps) => {
-              const DisplayComponent = isCreating
-                ? lazy(() => import('./new-account-wizard'))
```

Comment:
Why was this change made? This increases bundle size by about 0.2mb, which is significant. Would it be possible to maybe lazy-load the entire user-account-screen instead?

Reply:
Maybe. This lazy loading was causing an entire re-render of the account settings page when you edited a value.

Reply:
`UserAccountScreen` is already being lazy-loaded from `webapp-routes`, so something else is going on.

Reply:
`UserAccountScreen`, `NewAccountWizard` and `ExistingAccountDisplay` appear to be in the same bundle `6.bundle.js`, size 33kb.

Reply:
That's strange... might have been a caching issue on my end. Glad it's still being lazy loaded! Thanks for looking into it